src/common/reportsConfig.json

```diff
@@ -70,6 +74,29 @@
 		"applicationTableDataFields": [
 			"applicationId",
 			"orderId",
+			"status",
+			"amount"
+		]
+	},
+	"benefit_amounts":{
+		"autoGenerateFields": [
+			"serialNumber"
+		],
+		"applicationDataColumnDataFields": [
+			"otr",
+			"udid",
+			"disabilityType",
+			"studentType"
+		],
+		"calculatedAmountColumnDataFields": [
```

Comment:
These CSV labels should get generated runtime based on calculation rules as calculation rules may change scheme to scheme and could have any name @milindSansare